client/src/EstimatesComparison/scheme.js

```diff
@@ -142,9 +142,7 @@ const get_keys_in_sups = (include_stat) =>
     .value();
 
 const calculate_percent_value = (current_value, comparison_value) => {
```

Comment:
Fine if `current_value` is never undefined, but will return a `NaN` if it is. Maybe add a check for that or just an `|| 0`  on the default return case?